src/components/ExpandableCard.tsx

```diff
@@ -54,96 +52,40 @@ const ExpandableCard = ({
   }
 
   return (
-    <Accordion
-      border="1px"
-      borderColor="border"
-      borderRadius="sm"
-      display="flex"
-      flexDirection="column"
-      marginBottom="4"
-      cursor="pointer"
-      _hover={{
-        backgroundColor: "ednBackground",
-      }}
-      index={isVisible ? [0] : []}
-    >
-      <AccordionItem borderTopWidth="0" flex="1">
-        <Heading as="h3" m={0}>
-          <AccordionButton
-            width="100%"
-            px="6"
-            py="6"
-            flex="1"
+    <>
+      <Accordion type="single" collapsible className="mb-4">
+        <AccordionItem
+          value="item-1"
+          className="rounded-sm border border-gray-200/50 hover:bg-gray-100/50 dark:border-gray-600 dark:hover:bg-gray-800"
+        >
+          <AccordionTrigger
+            hideIcon
             onClick={onClick}
-            _hover={{
-              backgroundColor: "ednBackground",
-            }}
-            _expanded={{
-              backgroundColor: "transparent",
-            }}
+            className="w-full p-6 transition-colors hover:bg-gray-50 hover:text-black md:p-6 dark:hover:bg-gray-800 dark:hover:text-white [&[data-state=open]]:bg-transparent [&[data-state=open]]:text-black dark:[&[data-state=open]]:text-white"
           >
-            <Box
-              display="flex"
-              width="100%"
-              alignItems="center"
-              flexDir={{ base: "column", sm: "row" }}
-              textAlign="start"
-            >
-              <Box marginBottom={{ base: 2, sm: 0 }} me={{ base: 0, sm: 4 }}>
-                <Box
-                  display="flex"
-                  alignItems="center"
-                  sx={{
-                    svg: { me: "1.5rem" },
-                  }}
-                  my="4"
-                >
-                  {!!Svg && <Svg />}
-                  <Text fontSize="xl" fontWeight="semibold" flex="1" m="0">
-                    {title}
-                  </Text>
-                </Box>
-                <Text
-                  fontSize="sm"
-                  color="text200"
-                  marginBottom="0"
-                  width="fit-content"
-                >
+            <Flex className="w-full flex-col items-center text-left sm:flex-row">
+              <VStack className="items-center">
+                <HStack className="mb-2 mt-4">
+                  {Svg && <Svg className="mr-6" />}
+                  <h3 className="text-xl font-semibold">{title}</h3>
+                </HStack>
+                <p className="mb-0 w-fit text-sm text-gray-600">
```

Comment:
color change and unnecessary margin class in this case
```suggestion
                <p className="w-fit text-sm text-body-medium">
```